### PR TITLE
Lego 3550 badge legge til datafarger pa badges

### PR DESCRIPTION
**Pull request by TrygveUrdahl**
# PR Checklist
https://elvia.atlassian.net/wiki/spaces/TEAMATOM/pages/10427498683/Review+prosess

- [ ] Bumpet package?
- [ ] Updated changelog?
- [ ] Correct date in changelog?

## Describe PR briefly:
https://elvia.atlassian.net/browse/LEGO-3550

### CODE REVIEW

File: packages/components/components/elvis-badge/src/react/elvia-badge.types.tsx [hunk 1]
```diff
@@ -1,11 +1,19 @@
...
 import { ComponentPropsWithoutRef } from 'react';
 
-export type BadgeColor = 'red' | 'green' | 'neutral';
+type DataColors =
+  | 'green-apple'
+  | 'blue-berry'
+  | 'purple-plum'
+  | 'orange-mango'
+  | 'red-tomato'
+  | 'violet-grape';
```
[the remark this answers is not in the document]
Comment (TrygveUrdahl) [reply packages/components/components/elvis-badge/src/react/elvia-badge.types.tsx:10]: I see your point, and that wouldn't be a bad idea, however we already use these names so many other places. Tag for insance is very similar, and uses `e-tag--green-apple` and such

[the remark this answers is not in the document]
Comment (TomSchrier) [reply packages/components/components/elvis-badge/src/react/elvia-badge.types.tsx:10]: Implementing this with the literal color names would only make matters worse though

[the remark this answers is not in the document]
Comment (TrygveUrdahl) [reply packages/components/components/elvis-badge/src/react/elvia-badge.types.tsx:10]: Hmm fair enough, I can switch over to `data-1` etc.

**Review by TomSchrier — APPROVED** [review]
LGTM 🔔

**Pull request merged**